diff --git a/node_modules/@uns/crypto/dist/builders/uns-disclose-explicit.d.ts b/node_modules/@uns/crypto/dist/builders/uns-disclose-explicit.d.ts
index c95a189..48a5946 100644
--- a/node_modules/@uns/crypto/dist/builders/uns-disclose-explicit.d.ts
+++ b/node_modules/@uns/crypto/dist/builders/uns-disclose-explicit.d.ts
@@ -1,4 +1,4 @@
-import { Interfaces, Transactions } from "@arkecosystem/crypto";
+import { Interfaces, Transactions } from "@uns/ark-crypto";
 import { IDiscloseDemand, IDiscloseDemandCertification } from "../interfaces";
 export declare class UNSDiscloseExplicitBuilder extends Transactions.TransactionBuilder<UNSDiscloseExplicitBuilder> {
     constructor();
diff --git a/node_modules/@uns/crypto/dist/builders/uns-disclose-explicit.js b/node_modules/@uns/crypto/dist/builders/uns-disclose-explicit.js
index 303cf4c..e32342e 100644
--- a/node_modules/@uns/crypto/dist/builders/uns-disclose-explicit.js
+++ b/node_modules/@uns/crypto/dist/builders/uns-disclose-explicit.js
@@ -1,6 +1,6 @@
 "use strict";
 Object.defineProperty(exports, "__esModule", { value: true });
-const crypto_1 = require("@arkecosystem/crypto");
+const crypto_1 = require("@uns/ark-crypto");
 const enums_1 = require("../enums");
 class UNSDiscloseExplicitBuilder extends crypto_1.Transactions.TransactionBuilder {
     constructor() {
diff --git a/node_modules/@uns/crypto/dist/crypto/uns-crypto.js b/node_modules/@uns/crypto/dist/crypto/uns-crypto.js
index 8679492..fc3b84f 100644
--- a/node_modules/@uns/crypto/dist/crypto/uns-crypto.js
+++ b/node_modules/@uns/crypto/dist/crypto/uns-crypto.js
@@ -3,7 +3,7 @@ var __importDefault = (this && this.__importDefault) || function (mod) {
     return (mod && mod.__esModule) ? mod : { "default": mod };
 };
 Object.defineProperty(exports, "__esModule", { value: true });
-const crypto_1 = require("@arkecosystem/crypto");
+const crypto_1 = require("@uns/ark-crypto");
 const bytebuffer_1 = __importDefault(require("bytebuffer"));
 class UnsCrypto {
     signPayload(payload, passphrase) {
diff --git a/node_modules/@uns/crypto/dist/transactions/uns-disclose-explicit/uns-disclose-explicit.d.ts b/node_modules/@uns/crypto/dist/transactions/uns-disclose-explicit/uns-disclose-explicit.d.ts
index 2efab87..3da74c4 100644
--- a/node_modules/@uns/crypto/dist/transactions/uns-disclose-explicit/uns-disclose-explicit.d.ts
+++ b/node_modules/@uns/crypto/dist/transactions/uns-disclose-explicit/uns-disclose-explicit.d.ts
@@ -1,4 +1,4 @@
-import { Transactions, Utils } from "@arkecosystem/crypto";
+import { Transactions, Utils } from "@uns/ark-crypto";
 import ByteBuffer from "bytebuffer";
 export declare class DiscloseExplicitTransaction extends Transactions.Transaction {
     static typeGroup: number;
diff --git a/node_modules/@uns/crypto/dist/transactions/uns-disclose-explicit/uns-disclose-explicit.js b/node_modules/@uns/crypto/dist/transactions/uns-disclose-explicit/uns-disclose-explicit.js
index 56aaf30..d1c5e51 100644
--- a/node_modules/@uns/crypto/dist/transactions/uns-disclose-explicit/uns-disclose-explicit.js
+++ b/node_modules/@uns/crypto/dist/transactions/uns-disclose-explicit/uns-disclose-explicit.js
@@ -3,7 +3,7 @@ var __importDefault = (this && this.__importDefault) || function (mod) {
     return (mod && mod.__esModule) ? mod : { "default": mod };
 };
 Object.defineProperty(exports, "__esModule", { value: true });
-const crypto_1 = require("@arkecosystem/crypto");
+const crypto_1 = require("@uns/ark-crypto");
 const bytebuffer_1 = __importDefault(require("bytebuffer"));
 const enums_1 = require("../../enums");
 const schema_1 = require("./schema");
